src/share/content_renderer.js

```diff
@@ -83,16 +93,17 @@ function getContent(note) {
             content = `<button type="button" onclick="location.href='api/notes/${note.noteId}/download'">Download file</button>`;
         }
     }
-    else if (note.type === 'book') {
-        content = getChildrenList(note);
-    }
     else {
-        content = '<p>This note type cannot be displayed.</p>' + getChildrenList(note);
+        content = '<p>This note type cannot be displayed.</p>';
     }
+    var child = getChildrenList(note);
+    content += child === '' ? '' : `<hr>${child}`;
```

Comment:
`getChildrenList()` calls were removed up until here so that they are included on every note